js-chat/rollup.config.mjs

```diff
@@ -0,0 +1,24 @@
+import pkg from "./package.json" assert { type: "json" }
+import terser from "@rollup/plugin-terser"
+// import commonjs from "@rollup/plugin-commonjs"
+
+export default [
+  {
+    input: "../build/dist/js/productionLibrary/pubnub-chat.mjs",
+    external: ["pubnub", "format-util"],
+    output: [
+      {
+        file: pkg.main,
+        format: "cjs",
+      },
+      {
+        file: pkg.module,
+        format: "esm",
+      },
+    ],
+    plugins: [
+      terser()
+//       commonjs()
```

Comment:
This comment is related to content of `fixtures` folder. What is content of `fixtures` folder used for?

Reply:
these are files that are used by JS tests
the tests are not currently run automatically, but they are functional and can be run manually. I'm planning to include them in the CI process, but want to do it carefully so it doesn't make our build times even longer